lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -488,6 +488,7 @@ def _prepare_environment(self, environment: Environment, log: Logger) -> bool:
         # covert to azure node space, so the azure extensions can be loaded.
         for req in nodes_requirement:
             self._load_image_features(req)
+            self._check_image_capability(req)
```

Comment:
It should be called in `_check_image_capability`, because image features are updated there.

Reply:
Done